Runtime/Helpers/EnumHelper.cs

```diff
@@ -46,5 +46,29 @@ T max_inclusive
 
             return ( T )array.GetValue( UnityEngine.Random.Range( min_index, max_index + 1 ) );
         }
+
+        public static T Clamp<T>(
+            T value,
+            T min,
+            T max
+            ) where T : Enum
```

Comment:
Might as well actually define it for any `IComparable` although you'd need to check if it creates garbage (doubt it since it's generics).